JASP-R-Interface/jaspResults/DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: jaspResults
 Type: Package
 Title: Easy results for your JASP analysis
-Version: 1.12
+Version: 1.13
 Date: 2018-12-07
```

Comment:
Maybe the date should also be incremented occasionally?

Reply:
Possibly.........